package.json

```diff
@@ -3,6 +3,7 @@
   "description": "Material Components React",
   "license": "Apache-2.0",
   "scripts": {
+    "start": "webpack-dev-server --config test/screenshot/webpack.config.js --content-base test/screenshot",
```

Comment:
I don't think we can do the --content-base option here. I think we need the webpack.config.js file to live in the root. I don't think it transform the files in the packages dir. Try importing a React Component into the screenshot tests, and I don't think you'll get JSX errors.

Reply:
nope!